etc/prepare-tests.sh

```diff
@@ -10,12 +10,13 @@ mkdir -p build/test-reports/
 # Set up JSHint configuration for the generated sync functions
 cp "etc/jshintrc-sync-function-template.json" "$outputDir/.jshintrc"
 
-echo "Linting modules and specs with JSHint\n"
-node_modules/jshint/bin/jshint src/*.js test/*.js
+echo "Linting modules and specs with JSHint...\n"
+node_modules/jshint/bin/jshint src test/*.js
```

Comment:
Specifying the `src` directory as the target directly ensures that JSHint recurses into subdirectories.